onnxruntime/core/mlas/inc/mlas.h

```diff
@@ -1034,7 +1034,8 @@ MLASCALL
 MlasConvertHalfToFloatBuffer(
     const unsigned short* Source,
     float* Destination,
-    size_t Count
+    size_t Count,
+    bool useAVX
```

Comment:
why do we need this parameter?

Reply:
Hi Yufeng! I just applied this suggestion, let me know if it's any other change needed